src/js/node/http2.ts

```diff
@@ -1,10 +1,113 @@
 // Hardcoded module "node:http2"
+
+const { isTypedArray } = require("node:util/types");
+
 // This is a stub! None of this is actually implemented yet.
 const { hideFromStack, throwNotImplemented } = require("$shared");
 
-function connect() {
-  throwNotImplemented("node:http2 connect", 887);
-}
+const tls = require("node:tls");
+const net = require("node:net");
+const bunTLSConnectOptions = Symbol.for("::buntlsconnectoptions::");
+type Http2ConnectOptions = { settings?: Settings; protocol?: "https:" | "http:"; createConnection?: Function };
+const TLSSocket = tls.TLSSocket;
+const EventEmitter = require("node:events");
+const { Duplex } = require("node:stream");
+const primordials = require("internal/primordials");
+
+const { H2FrameParser, getPackedSettings, getUnpackedSettings } = $lazy("internal/http2");
+
+const sensitiveHeaders = Symbol.for("nodejs.http2.sensitiveHeaders");
```

Comment:
Do these symbols need to be public?

Reply:
dont need to be public, can I private this only for builtins?

Reply:
If they do not need to be public, I suggest adding them to a separate internal module which is loaded elsewhere.

Reply:
sensitiveHeaders header is the only public one